central/declarativeconfig/manager_impl.go

```diff
@@ -68,5 +71,21 @@ func (m *managerImpl) WatchDeclarativeConfigDir() {
 // TODO(ROX-14693): Add upserting transformed resources.
 // TODO(ROX-14694): Add deletion of resources.
 func (m *managerImpl) ReconcileDeclarativeConfigs(contents [][]byte) {
+	configurations, err := declarativeconfig.ConfigurationFromRawBytes(contents...)
+	if err != nil {
+		log.Errorf("Error during unmarshalling of declarative configuration files: %+v", err)
+		return
+	}
+	transformedConfigurations := make(map[reflect.Type][]proto.Message, len(configurations))
+	for _, configuration := range configurations {
+		transformedConfig, err := m.t.Transform(configuration)
```

Comment:
nit: perhaps we can rename `t` to `transform` or `transformer`?`m.t` seems kind of confusing